.gitignore

```diff
@@ -12,5 +12,3 @@ yarn-error.log*
 
 # Unit test / coverage reports
 coverage/
-
-docs/
```

Comment:
Note to other devs: although this is checked in, we can ignore anything in docs/ during PR review.